docs/roman/references_general/gain_reffile.inc

```diff
@@ -0,0 +1,45 @@
+GAIN reference file
+-------------------
+
+:REFTYPE: GAIN
+:Data model: `~jwst.datamodels.GainModel`
+
+The GAIN reference file contains a pixel-by-pixel gain map, which can be
+used to convert pixel values from units of DN to electrons. The gain
+values are assumed to be in units of e/DN.
+
+.. include:: ../references_general/gain_selection.inc
+
+.. include:: ../includes/standard_keywords.inc
+
+Type Specific Keywords for GAIN
++++++++++++++++++++++++++++++++
+In addition to the standard reference file keywords listed above,
+the following keywords are *required* in GAIN reference files,
+because they are used as CRDS selectors
+(see :ref:`gain_selectors`):
+
+===============  ==============================
+Keyword          Data Model Name
+===============  ==============================
+DETECTOR         model.meta.instrument.detector
+SUBARRAY         model.meta.subarray.name
+BUNIT\ :sup:`1`  model.meta.bunit_data
+===============  ==============================
+
+:sup:`1` BUNIT is not used as a CRDS selector, but is required in the
+"SCI" extension header of GAIN reference files to document the units
```

Comment:
I think we can drop BUNIT.

Reply:
Can we also drop SUBARRAY ?

Reply:
This is all removed.. if you look at this file, it is clear that I missed editing it.. it even has a jwst reference in it!